app/client/components/contactUs/contactUs.tsx

```diff
@@ -159,10 +159,38 @@ export const ContactUs = (props: ContactUsProps) => {
     }
   };
 
+  const HeaderTitle = (
+    <span>
+      Need to contact us{" "}
```

Comment:
What does the `{" "}` do?

Reply:
Hey Kelvin ... it forces a space between `Need to contact us` and `about something?` on desktop and higher breakpoints (the "about something?" part is only visible for the larger breakpoints)

Reply:
I see. Thanks!